recipes/cgns/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "cgns" %}
+{% set version = "3.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name|upper }}/{{ name|upper }}/archive/v{{ version }}.tar.gz
+  sha256: 677d0da94f0892b72b96da7cf4de51f661acfff1141c585af50a531fa161773f
+  patches:
+    - if-block.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - make
+    - zlib
+    - hdf5 =1.8
```

Comment:
zlib and hdf5 should be host requirements

Reply:
FYI, if you don't build hdf5 with zlib then it is not required by CGNS. CGNS does not have compression (via HDF5).

Reply:
Fixed with 439d212.

Reply:
@brtnfld Thanks for the suggestion. I think conda-forge's hdf5 builds with zlib enabled -- here's its [build script](https://github.com/conda-forge/hdf5-feedstock/blob/master/recipe/build.sh).